pkg/loader/load.go

```diff
@@ -771,7 +852,7 @@ func (b *batchManager) execAccumulatedDMLs() (err error) {
 
 func (b *batchManager) execDDL(txn *Txn) error {
 	if err := b.fExecDDL(txn.DDL); err != nil {
-		if !pkgsql.IgnoreDDLError(err) {
+		if b.destDBType == "oracle" || !pkgsql.IgnoreDDLError(err) {
```

Comment:
Why?

Reply:
because pkgsql.IgnoreDDLError do not support oracle error. so if it found error, error will be return for oracle

Reply:
don't understand. If `pkgsql.IgnoreDDLError` do not support oracle error, all errors will be returned.